NBXplorer/Properties/launchSettings.json

```diff
@@ -0,0 +1,45 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:4771/",
+      "sslPort": 0
+    }
+  },
+  
+  "profiles": {
+    "NBXplorer - BTC on Mainnet": {
+      "commandName": "Project",
+      "commandLineArgs": "--btcrpcuser=myuser --btcrpcpassword=mypassword",
```

Comment:
remove those, by default BTCPay can use cookie authentication so that you don't need to setup rpc password and user.

Reply:
These are needed for the connection between the btc/ltc/dash node and NBXplorer, not the NBXlorer and its API client.

If I remove them I get the 401 error:
![image](https://user-images.githubusercontent.com/910321/64038142-a9085680-cb57-11e9-8ba5-9834e6f342cd.png)

Reply:
@andrasfuchs this is because you configured rpcuser/password in your bitcoind configuration file. Remove this and it should work.

Reply:
@NicolasDorier I removed the following two lines from my `bitcoin.conf`:
```
rpcuser=myuser
rpcpassword=mypassword
```

I started the Bitcoin Core Qt with `-server` and I started NBXplorer without the `--btcrpcuser=myuser --btcrpcpassword=mypassword` parameters, but I still got the 401 (Unathorized) error.

What could I possible do wrong?

Reply:
@andrasfuchs run `bitcoind` instead.

Reply:
It works with `bitcoind`. 
Should we not put this info into the README.md as well? It's not that obvious if you ask me :)

Reply:
yes I think documenting this in README is good.

Reply:
Ok, I updated the readme as well.